builtin/providers/aws/resource_aws_emr_cluster.go

```diff
@@ -253,12 +258,16 @@ func resourceAwsEMRClusterCreate(d *schema.ResourceData, meta interface{}) error
 
 		ReleaseLabel:      aws.String(d.Get("release_label").(string)),
 		ServiceRole:       aws.String(d.Get("service_role").(string)),
+		AutoScalingRole:   aws.String(d.Get("autoscaling_role").(string)),
```

Comment:
This defined here and also, on Line 268

The declaration on line 268 is the correct one as it's optional and cannot be guaranteed to be here. We would be passing an empty string to the API with this declaration and no autoscaling role being defined